integration-tests/src/test/java/com/datastax/oss/driver/mapper/model/inventory/ProductDao.java

```diff
@@ -32,4 +33,10 @@
 
   @SetEntity
   void set(Dimensions dimensions, UdtValue udtValue);
+
+  @Insert
+  void save(Product product);
+
+  @Insert(customClause = " USING TIMESTAMP :timestamp")
```

Comment:
is this white space before `USING` needed? If yes I think that it may be error-prone and we should take 
`"USING ...` as an argument and take care of inserting whitespace.

Reply:
No it's not needed, the processor code does indeed add a whitespace. That's a typo, I'll remove the whitespace here.